msteams-platform/bots/what-are-bots.md

```diff
@@ -12,8 +12,10 @@ A bot is also referred to as a chatbot or conversational bot. It is an app that
 
 > [!IMPORTANT]
 > Currently, bots are available in Government Community Cloud (GCC) and GCC-High but not in Department of Defense (DOD).
-> 
-> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/channel-connect-teams).
+>
+> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/how-to-deploy-gov-cloud-high) and bot channel registration should be done in Azure Government portal.
```

Comment:
We should also talk about or add a link to consent modeling in GCCH. ie tenants are responsible for enabling LOB applications and how the data is processed / where it is stored.